build.gradle

```diff
@@ -23,6 +23,7 @@ ext.bndWorkspace = new Workspace(rootDir, bnd_cnf)
 if (bndWorkspace == null) {
   throw new GradleException("Unable to load workspace ${rootDir}/${bnd_cnf}")
 }
+bndWorkspace.setOffline(project.gradle.startParameter.offline)
```

Comment:
Why are you doing it here? I would think this change needs to be isolated to the bnd plugin rather than requiring everyone edit their root project build.gradle file.